com.unity.render-pipelines.universal/Shaders/BakedLit.shader

```diff
@@ -69,6 +69,9 @@ Shader "Universal Render Pipeline/Baked Lit"
             #include "Packages/com.unity.render-pipelines.universal/Shaders/BakedLitInput.hlsl"
             #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl"
 
+            // Force enable fog fragment shader evaluation
+            #define _FOG_FRAGMENT 1
```

Comment:
If _FOG_FRAGMENT is already defined in shader library why do we need to redefine here?

Reply:
We don't if it doesn't include the library (might not be the case, but perhaps it then just should).
Most likely it's just leftovers when I had them per shader. I'll double check it.